### ISSUE TITLE
Staging Review finding: Cards under Submissions heading level

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#114654]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing
**Product:** Accredited Rep Portal
**Feature:** Claims Form Upload

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that meets best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

The Submissions heading is an h2, but the cards underneath them are also h2. 

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/representative/submissions

<img width="760" height="638" alt="Image" src="https://github.com/user-attachments/assets/6791ae32-7cce-4c58-87f4-1075d182ed13" />

</details>

## Recommended action

Since the cards are a subsection of Submissions, they should be an h3.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jquispe-oddball** [comment]
@candilemoine @lauraparadis-nava this is on staging

<img width="1063" height="609" alt="Image" src="https://github.com/user-attachments/assets/28291ca5-436b-490c-9c33-445b04ef73f9" />

**Comment by candilemoine** [comment]
@candilemoine will QA

**Comment by candilemoine** [comment]
LGTM👍🏻
